policyengine_canada/variables/household/income/household/household.refundable_tax_credits.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_canada.model_api import *
+
+
+class household_refundable_tax_credits(Variable):
```

Comment:
rename the file `household_refundable_tax_credits.py` (there's an extra `.` in there)